propslab/src/components/donations/DonationsDisplay.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+const DonationsDisplay = (props) => {
```

Comment:
This would take in a component that renders each of the donations, or it would be a component that takes in a donation object to display a donation.